pkg/client/jiva/model.go

```diff
@@ -21,24 +21,28 @@ const (
 	snapSuffix         = ".img"
 )
 
+// Volumes
```

Comment:
There is no description ?!

Reply:
Yeah. So I wasn't really sure about the description part and didn't want to add wrong description. So avoided it.
I'll try to go through the code again and add descriptions